src/test/kotlin/ts2kt/RunnerTest.kt

```diff
@@ -1,47 +0,0 @@
-package ts2kt
-
-import node.EncodingOptions
-import node.fs
-import kotlin.test.Test
-import kotlin.test.assertEquals
-
-class RunnerTest {
-    val TS_EXT = ".ts"
-    val KT_EXT = ".kt"
-    val UNVERIFIED_FILE_PREFIX = "// OUT:"
-
-    @Test
```

Comment:
Why go from using Kotlin tests to javascript tests?

Reply:
Hi Eric! For a while we've decided to make a step back and have javascript tests because in foreseeable future the build process will possibly undergo some noticeable changes and  the simpler the starting point of this journey will be - the better.

Reply:
Thanks.